openapi/p4pa-pu-bff.openapi.yaml

```diff
@@ -1162,3 +1196,26 @@ components:
         number:
           type: integer
           format: int64
+    DebtPositionDetailDTO:
+      type: object
+      required:
+        - debtor
+        - debtPositionTypeOrgDescription
+        - debtPositionTypeOrgCode
+        - status
+        - paymentOptions
+      properties:
+        debtor:
+          $ref: '#/components/schemas/PersonDTO'
```

Comment:
typeMap also this: is the current PersonDTO documented on this openApi really used? could it be possible to typeMap also that?